### ISSUE TITLE
nan loss when training auto encoder

### ISSUE THREAD

**Issue by jeongHwarr** [zoomin-lee/SemCity#8]
Thank you for your impressive project!
I want to dry run this project, and I tried train autoencoder with kitti dataset.
However, loss of autoencoder is nan when trining. 
I don't know why the loss is nan. 

One thing that is suspicious is that warning messages about lr_scheduler.step and optimizer.step appear.
I used the environment you recommended. (torch 2.1)
I want to know how to make an autoencoder train successfully.

![image](https://github.com/zoomin-lee/SemCity/assets/16534413/83a9fb0a-cbee-4008-bc65-44baf2734a05)
